tests/E2E/test/campaigns/common_scenarios/module.js

```diff
@@ -8,7 +9,16 @@ module.exports = {
   installModule: function (client, ModulePage, AddProductPage, moduleTechName) {
     test('should go to "Module" page', () => client.goToSubtabMenuPage(Menu.Improve.Modules.modules_menu, Menu.Improve.Modules.modules_services_submenu));
     test('should set the name of the module in the search input', () => client.waitAndSetValue(ModulePage.module_selection_input, moduleTechName));
-    test('should click on "Search" button', () => client.waitForExistAndClick(ModulePage.selection_search_button));
+    test('should click on "Search" button', () => {
+      return promise
+        .then(() => client.waitForExistAndClick(ModulePage.selection_search_button))
+        .then(() => client.isVisible(AddProductPage.symfony_toolbar, 3000))
```

Comment:
I've added an environment variable to make the debug mode disabled, isn't a better solution than click on Symfony debug toolbar to hide it?

Reply:
You may create `defines_custom.inc.php` file in `config` folder and set `_PS_MODE_DEV_` to false /c @mbadrani

Reply:
ping @fatmaBouchekoua wdyt about my idea? it's better than click on debug toolbar to hide it on every e2e test, isn't it?

Reply:
I'm really agree with you, I think the Symfony debug toolbar must be disable while running e2e tests.

Reply:
These tests can be launched on local, and if we will disable the debug mode, developers will not have the choice to run them with the debug mode enabled.
So I think that we have to hide the Symfony debug toolbar if it's visible in some cases (only if that toolbar is hiding a button)

Reply:
Every user/developer can disable debug mode, you may also add a check in e2e to stop the test suite and inform the developer that the test suite is only available for a shop in production mode.

Not only you don't need to rely on DOM to disable the debug toolbar but the test suite will run faster as there is a really big difference of performance between dev and production mode (~150% on some benchmarks).